lib/travis/logs/aggregate.rb

```diff
@@ -19,15 +19,23 @@ def setup
       def run
         loop do
           aggregate_logs
-          sleep Travis.config.logs.intervals.vacuum
+          sleep sleep_interval
         end
       end
 
       def aggregate_logs
-        Travis::Logs::Services::AggregateLogs.run
+        aggregator.run
       rescue Exception => e
         Travis::Exceptions.handle(e)
       end
+
+      private def aggregator
```

Comment:
Small style nit, putting `private` in front of individual methods is typically non-idiomatic for ruby which favors them done as a group ala:

```
private 

def aggregator 
......
end

def sleep_interval
.....
end
```